component/site/libraries/commonfunctions.php

```diff
@@ -420,43 +448,34 @@ public static function sendAdminMail( $adminName, $adminEmail, $subject='', $tit
 			$mail->addReplyTo($adminEmail);
 		}
 
-                // JEvents category admin only or both get notifications
-                if ($params->get("com_notifyboth",0)==0 || $params->get("com_notifyboth",0)==1){
-                    $mail->addRecipient($adminEmail);
-                    if ($params->get("com_notifyboth",0)==1){
-                            $jevadminuser = new  JUser($params->get("jevadmin",62));
-                            if ($jevadminuser->email != $adminEmail){
-                                    $mail->addCC($jevadminuser->email);
-                            }
-                    }
-                }
-                // Just JEvents admin user
-                else if ($params->get("com_notifyboth",0)==2){
-                    $jevadminuser = new  JUser($params->get("jevadmin",62));
-                    if ($jevadminuser->email != $adminEmail){
-                            $mail->addRecipient($jevadminuser->email);
-                    }                    
-                }
+		if ($add_cc > '') {
```

Comment:
$add_cc may not be declared so should the test should be:

if (isset($add_cc) && $add_cc  != "") {